cli/generateAddressCmd.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	archethic "github.com/archethic-foundation/libgo"
+	"github.com/spf13/cobra"
+)
+
+func GetGenerateAddressCmd() *cobra.Command {
+	generateAddressCmd := &cobra.Command{
+		Use:   "generate-address",
+		Short: "Generate address",
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I think also the output should be the address without additional data, hence we could copy or use this as input for other function.